trestle/core/commands/remove.py

```diff
@@ -15,10 +15,113 @@
 # limitations under the License.
 """Trestle Remove Command."""
 
+import pathlib
+import warnings
+
 from ilcli import Command  # type: ignore
 
+import trestle.core.const as const
+import trestle.core.err as err
+from trestle.core import utils
+from trestle.core.models.actions import CreatePathAction, RemoveAction, WriteFileAction
+from trestle.core.models.elements import Element, ElementPath
+from trestle.core.models.file_content_type import FileContentType
+from trestle.core.models.plans import Plan
+from trestle.utils import fs
+
 
 class RemoveCmd(Command):
     """Remove a subcomponent to an existing model."""
 
     name = 'remove'
+
+    def _init_arguments(self):
+        self.add_argument(
+            f'-{const.ARG_FILE_SHORT}',
+            f'--{const.ARG_FILE}',
+            help=const.ARG_DESC_FILE + ' to remove component/subcomponent to.',
+            required=True
+        )
+        self.add_argument(
+            f'-{const.ARG_ELEMENT_SHORT}',
+            f'--{const.ARG_ELEMENT}',
+            help=const.ARG_DESC_ELEMENT + ' to remove.',
+            required=True
+        )
+
+    def _run(self, args):
```

Comment:
If you could add some type definitions to the function definitions it would be fantastic.
e.g. 
```python
def _run(self, args: argparse.Namespace) -> int: 

```
A further note here - this function can pass back a return code. To date we have been using 0 for pass and 1 for any error (e.g. errors are untyped).

Reply:
In reality this happens already to some extent by exception - but testing this function specifically allows you to check that exceptions are not been thrown (as opposed to sys.exit()).

Reply:
Done, added.